MultiPartyExample.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 51;
+	objectVersion = 50;
```

Comment:
Intentional right?

Reply:
I mean... I didn't change it... Xcode did. But it all still compiles happily.